### ISSUE TITLE
Docs state that `options` is an optional param to `parseCurrency`, but Typescript definitions declare it as required.

### ISSUE THREAD

**Issue by ascott18** [dm4t2/vue-currency-input#93]
As the title states, the Typescript definitions declare 

``` ts

export function parseCurrency (formattedValue: string, options: CurrencyInputOptions): number

declare module 'vue/types/vue' {
  interface Vue {
    $parseCurrency (formattedValue: string, options: CurrencyInputOptions): number
  }
}

```

But if the docs are to believed, this should be
``` ts

export function parseCurrency (formattedValue: string, options?: CurrencyInputOptions): number

declare module 'vue/types/vue' {
  interface Vue {
    $parseCurrency (formattedValue: string, options?: CurrencyInputOptions): number
  }
}

```

There's also a typo in the docs ("und defaults")
![image](https://user-images.githubusercontent.com/5017521/76127587-e2ae9400-5fb6-11ea-9272-57361e540f82.png)

**Comment by dm4t2** [comment]
Thanks for reporting 👍

**Comment by dm4t2** [comment]
Fixed with Release [1.17.2](https://github.com/dm4t2/vue-currency-input/releases/tag/1.17.2) 🎉
